whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapExceptionMapper.java

```diff
@@ -42,6 +42,13 @@ public Response toResponse(final Exception exception) {
             return createErrorResponse(HttpServletResponse.SC_BAD_REQUEST, "400 Bad Request",
                     "unknown " + parameterName);
```

Comment:
Ideally if we are throwing a custom exception then we should catch all exception in service layer and turn it into an RdapException, I am not sure why we still have individual exception here but feel free to leave it as is it if it is a complicated or big change

Reply:
discussed via Mattermost
There are some exception that we can not control or take over with RdapException